timepiece/entries/lookups.py

```diff
@@ -11,9 +11,11 @@ class ActivityLookup(ModelLookup):
 
     def get_query(self, request, term):
         results = super(ActivityLookup, self).get_query(request, term)
-        project = Project.objects.get(pk=request.GET.get('project', ''))
-        if project and project.activity_group:
-            return project.activity_group.activities.all()
+        project_pk = request.GET.get('project', None)
+        if project_pk not in [None, '']:
+            project = Project.objects.get(pk=project_pk)
+            if project and project.activity_group:
+                return project.activity_group.activities.all().filter(name__startswith=term)
```

Comment:
should this be `name_icontains` instead of `name_startswith`?

It would also be great to add a test that specifically tests the filtered results.